app/assets/javascripts/discourse/tests/integration/modifiers/delayed-destroy-test.gjs

```diff
@@ -0,0 +1,58 @@
+import { renderSettled } from "@ember/renderer";
```

Comment:
I think you might need to use `settled` from `"@ember/test-helpers"` here instead? I could be wrong though

Reply:
using `settled` was my initial plan, but it didn't work out, so I reached out to the Ember community for help:
* https://discuss.emberjs.com/t/testing-interim-states-with-the-latest-test-helpers/15117/5?u=yuriyk
* https://discord.com/channels/480462759797063690/1353694007385129021

and the official docs: https://api.emberjs.com/ember/release/classes/renderer/methods/?anchor=renderSettled

Reply:
Totally fair, love that you got an answer from the Ember community :)